cypher/cypher-docs/src/docs/dev/ql/administration/security/index.asciidoc

```diff
@@ -24,11 +24,13 @@ This section explains how to use Cypher to manage Neo4j role-based access contro
 * <<administration-security-subgraph, Graph and sub-graph access control>>
 ** <<administration-security-subgraph-introduction, The `GRANT`, `DENY` and `REVOKE` commands>>
 ** <<administration-security-subgraph-show, Listing privileges>>
-** <<administration-security-subgraph-traverse, The `TRAVERSE` privilege>>
-** <<administration-security-subgraph-read, The `READ` privilege>>
-** <<administration-security-subgraph-match, The `MATCH` privilege>>
-** <<administration-security-subgraph-write, The `WRITE` privilege>>
 ** <<administration-security-subgraph-revoke, The `REVOKE` command>>
+* <<administration-security-reads, Fine-grained reads>>
+** <<administration-security-reads-traverse, The `TRAVERSE` privilege>>
+** <<administration-security-reads-read, The `READ` privilege>>
+** <<administration-security-reads-match, The `MATCH` privilege>>
+* <<administration-security-writes, Fine-grained writes>>
```

Comment:
I'm not sure I like Fine-grained read / writes. Should we just call them Read privileges and Write privileges as they are both broad like GRANT READ and fine grained.

Reply:
I'm inclined to agree - simplifying the structure of the administration section will be addressed in the coming week, as we plan to clean up the architecture a bit before 4.1 goes out.

Naming the groupings "Read privileges" and "Write privileges" would keep things simple, and enable us to move towards a structure that groups the privileges as follows:
* Managing privileges  
** Managing graph privileges
** Managing database privileges
** Managing role privileges

For the context of this PR though, I'm happy to rename the new sections when we come to that work, if that is helpful.

Reply:
Ok, I also like read privileges and write privileges better! Will update that in this PR since I think there will be more places to update (links etc.) if we wait with it.